reporting.md

```diff
@@ -29,70 +31,60 @@ During installation, you will be prompted to log in with a user account. The acc
 
 ### Running the installer
 - From the [Simeon portal](https://app.simeoncloud.com/), click **Install** on the navigation pane > select the **Install Power BI Reporting** tab
-- Enter the domain name of the tenant > **INSTALL** > authenticate with an account that meets the [prerequisites](#prerequisites) > when prompted, select the Azure Subscription and desired location for the SQL server
+- Enter the domain name of the tenant > **INSTALL**
+- Authenticate with an account that meets the [prerequisites](#prerequisites)
+<br />
+<img src="https://raw.githubusercontent.com/simeoncloud/docs/feature/loganalytics/assets/images/power_bi_auth_to_tenant.png" width: 338; height: auto;/>
+- When prompted, select the Azure Subscription
+<br />
+<img src="https://raw.githubusercontent.com/simeoncloud/docs/feature/loganalytics/assets/images/power_bi_subscription.png" width: 337; height: auto;/>
+- When prompted, select an Azure location
+<br />
+<img src="https://raw.githubusercontent.com/simeoncloud/docs/feature/loganalytics/assets/images/power_bi_location.png" width: 337; height: auto;/>
 - Once the installation is complete, click **Run Backfill Now**. This will backfill your Power BI report with the past 72 hours of Sync data.
+<br />
+<img src="https://raw.githubusercontent.com/simeoncloud/docs/feature/loganalytics/assets/images/power_bi_install_complete.png" width: auto; height: auto;/>
+
+### Setup the Power BI Datasets
+- Go to [Power BI](https://app.powerbi.com) > Workspaces > Simeon Cloud > click the three dots next to **Baseline Compliance Report** Dataset > Settings > **Take Over** > **Take Over**
```

Comment:
the take over is here twice intentionally, right?

Reply:
Correct, you hit Take Over and it opens a new popup where you have to confirm by hitting Take Over again.